etc/profile-a-l/git-cola.profile

```diff
@@ -0,0 +1,65 @@
+# Firejail profile for git-cola
+# Description: Linux native frontend for Git
+# This file is overwritten after every install/update
+# Persistent local customizations
+include git-cola.local
+# Persistent global definitions
+include globals.local
+
+ignore noexec ${HOME}
+
+noblacklist ${HOME}/.gitconfig
+noblacklist ${HOME}/.git-credentials
+noblacklist ${HOME}/.gnupg
+noblacklist ${HOME}/.ssh
+noblacklist ${HOME}/.config/git
+noblacklist ${HOME}/.config/git-cola
+# Put your editor,diff viewer config path below and uncomment to load settings
+# noblacklist ${HOME}/
+
+include allow-python2.inc
+include allow-python3.inc
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
```

Comment:
include disable-xdg.inc ?? How likely is it a user would need access to Documents, Music, Pictures, or Videos?

Reply:
Thanks. I was testing with files in Documents and forgot to add it.